pumps/prometheus.go

```diff
@@ -245,6 +248,11 @@ func (p *PrometheusPump) WriteData(ctx context.Context, data []interface{}) erro
 		default:
 		}
 		record := item.(analytics.AnalyticsRecord)
+
+		if !(p.conf.TrackAllPaths || record.TrackPath) {
+			record.Path = "unknown"
```

Comment:
Just a small comment: Setting a const here will make it easier to read and avoid typos if we need to work on this again in the future. I'd set a UnknowPath const and use it here and in the unit test, wdyt?